samples-aspnetcore-3x/self-hosted-login/okta-aspnetcore-mvc-example/Views/Shared/_Layout.cshtml

```diff
@@ -6,6 +6,7 @@
     <title>@ViewData["Title"] - okta_aspnetcore_mvc_example</title>
     <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
     <link rel="stylesheet" href="~/css/site.css" />
+    @RenderSection("head", false)
```

Comment:
What does this line do?

Reply:
It ensures that the html elements defined in a "head" section is rendered in the head: https://github.com/okta/samples-aspnetcore/pull/51/files#diff-31af1060bf9f469f51cc2690670de949db84c9749ee7a652d5772002db4626d5R6